resources/views/selectedfilm.blade.php

```diff
@@ -19,6 +20,10 @@
       </div>
       <div class="card-body justify-content-center">
         <p>  {{$body->overview}}<p>
+        <p>  {{$body->vote_average}}<p>
```

Comment:
Maybe we can add text or some kind of indicator here to clarify that what this is for the user.

Reply:
Yet again animus saves the day. Very true, will rectify and add some sort of text to that explains the numbers